src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -41,4 +44,15 @@ internal class OktaSessionStorage : IOktaSessionStorage {
 		}
 		return new();
 	}
+	private void WriteTextToFile( string path, string Content ) {
```

Comment:
nit: we usually separate methods with empty lines
```suggestion

	private void WriteTextToFile( string path, string Content ) {
```